internal/cli/atlas/customdbroles/list.go

```diff
@@ -79,8 +80,8 @@ func ListBuilder() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().IntVar(&opts.PageNum, flag.Page, 0, usage.Page)
-	cmd.Flags().IntVar(&opts.ItemsPerPage, flag.Limit, 0, usage.Limit)
+	_ = cmd.Flags().MarkDeprecated(flag.Page, deprecatedFlagMessage)
+	_ = cmd.Flags().MarkDeprecated(flag.Limit, deprecatedFlagMessage)
 
```

Comment:
the flag needs to exists to be deprecated, otherwise it returns an error about the flag not existing 
```suggestion
	cmd.Flags().IntVar(&opts.PageNum, flag.Page, 0, usage.Page)
	cmd.Flags().IntVar(&opts.ItemsPerPage, flag.Limit, 0, usage.Limit)
	_ = cmd.Flags().MarkDeprecated(flag.Page, deprecatedFlagMessage)
	_ = cmd.Flags().MarkDeprecated(flag.Limit, deprecatedFlagMessage)

```

Reply:
Thank you!